webapp/backend/models.py

```diff
@@ -1,46 +1,116 @@
 from django.db import models
 
+"""
+OBJECTS
+"""
+
 class User(models.Model):
+    """
+    Stores information about every lambda-user.
+    id: the okeanos id of the user.
+    token: the okeanos token of the user.
+    """
     id = models.CharField("UUID", primary_key=True, null=False, blank=False, unique=True, default="", max_length=255, help_text="Unique user id asign by Astakos")
-    token = models.CharField("Okeanos Token", null=True, blank=True, unique=True, default="", max_length=64, help_text="Token provided by ~okeanos.")
 
 class Project(models.Model):
+    """
+    Stores information about every okeanos project that has been used with the LoD.
+    id: the okeanos id of the project.
+    description: a small description of the project.
+    """
     id = models.AutoField("Project ID", primary_key=True, null=False, blank=False, unique=True, default="", help_text="Project id provided by kamaki.")
-    description = models.TextField("Project Description", null=True, blank=True, unique=False, default="", help_text="The description of a project.")
+    description = models.TextField("Project Description", default="",
+                                   help_text="The description of a project.")
+
+
+class Server(models.Model):
+    """
+    Stores information about every server created for the LoD service.
+    id: the okeanos id of the server.
+    """
+    id = models.AutoField("Server ID", primary_key=True, null=False, blank=False,
+                          unique=True, default="",
+                          help_text="Server id provided by kamaki.")
 
 class Cluster(models.Model):
+    """
+    Stores every cluster created for the LoD service.
+    id: a unique identifier the service creates for every cluster.
+    :model: models.Server
+    :model: models.Cluster_info
+    """
     id = models.AutoField("Cluster ID", primary_key=True, null=False, help_text="Auto-increment cluster id.")
     # OneToOneField is a ForeignKey with unique=True. Django recommends using OneToOneField instead of a ForeignKey
     # with unique=True.
-    master_server = models.OneToOneField('Server', null=True, blank=True, on_delete=models.CASCADE)
-
-class Server(models.Model):
-    id = models.AutoField("Server ID", primary_key=True, null=False, blank=False, unique=True, default="", help_text="Server id provided by kamaki.")
-    hostname = models.CharField("Hostname", null=False, blank=False, unique=True, max_length=100)
-    public_ip = models.GenericIPAddressField("Public IP", null=False, blank=False, unique=True)
-    private_ip = models.GenericIPAddressField("Private IP", null=False, blank=False, unique=False)
+    master_server = models.OneToOneField(Server, null=True, blank=True, on_delete=models.CASCADE)
+    cluster_info = models.CharField('Cluster info', help_text="Cluster information in xml format.")
```

Comment:
Changed it to TextField item,seems better since it will keep a lot of characters.